llm/ext_server.go

```diff
@@ -0,0 +1,334 @@
+package llm
```

Comment:
`ggml` models would no longer run with this change. We probably want to send back an error here that notifies the user if we detect a `ggml` model: https://github.com/jmorganca/ollama/blob/main/llm/llm.go#L79

Here's a good error message: https://github.com/jmorganca/ollama/blob/37d95157dfed0726d751d3a694e9d6cd3c923097/server/routes.go#L106C5-L106C21